docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -401,6 +411,20 @@ Key Interfaces
 
         The EllipticCurvePublicKey object for this private key.
 
+    .. method:: sign(data, signature_algorithm)
+
+        .. versionadded:: 1.5
+
+        Sign one block of data which can be verified later by others using the
+        public key.
+
+        :param bytes data: The message string to sign.
+
+        :param signature_algorithm: An instance of a
+            :class:`EllipticCurveSignatureAlgorithm` provider.
```

Comment:
This is not consistent with the ``signer`` method documentation. Should I changed there too?